recipes/custom-headers.md

```diff
@@ -2,11 +2,11 @@
 
 _(Update: As of v0.9.0, users may set Custom Headers using the `headers` option)_
 
-Occasionally a development environment needs to mirror certain aspects of the production environment. That may mean adding custom headers to each request the server responds to. Thankfully, this is exceptionally straightforward using the [Koa](https://koajs.com) underpinnings and the [`middleware`](https://github.com/shellscape/webpack-plugin-serve#middleware) option.
+Occasionally a development environment needs to mirror certain aspects of the production environment. That may mean adding custom headers to each request the server responds to. On `webpack-plugin-serve` you have two ways to do that: the straightforward using the [Koa](https://koajs.com) underpinnings and our custom middleware. 
```

Comment:
The change to the start of the sentence changes some of the initial meaning -- in this version, it sounds like using `koa` is straightforward, but the middleware is not (in the old version, both are straightforward). I would either remove "the straightforward" from this version or leave the old version and similarly change the end to "our custom middleware".

also nit in this version: "On" -> "With"

Reply:
Those changes are actually improving my English!! Thank you!! It is nice to have a well written, concise and clear documentation!! Thank you for your improvements!!